ci/run_tests.sh

```diff
@@ -126,5 +126,9 @@ pushd preprocess_hello
 "$fpm" build
 popd
 
+pushd cpp_files
+"$fpm" build
```

Comment:
Let's run the test executable to check that it links correctly.
```suggestion
"$fpm" test
```